README.md

```diff
@@ -21,7 +21,14 @@ This README documents the current strawman architecture.  Changes must be made v
     2. ARAs respond to ARS with Message
     3. ARA sends query messages to KPs
     4. KPs respond to ARAs with Message
-3. All communication between components conforms to the ReasonerAPI Message spec
+3. Interfaces:
+    1. All communication between the ARS and ARAs conforms to the ReasonerAPI Message spec
+    2. KP can expose their information using these methods:
+       A. ReasonerAPI Message
+       B. Any SmartAPI-annotated interface
+       C. A text-file dump conforming to KGX standards
```

Comment:
Will these always be _text_ files? A more compact format might be desirable at some point.

Reply:
Great question, I suppose not.  I'll modify to reflect that.